python/cog/server/http.py

```diff
@@ -104,20 +93,37 @@ async def healthcheck_startup_failed() -> Any:
         )
 
 
-def create_app(  # pylint: disable=too-many-arguments,too-many-locals,too-many-statements
+def create_app(
     config: CogConfig,  # pylint: disable=redefined-outer-name
     shutdown_event: Optional[threading.Event],  # pylint: disable=redefined-outer-name
     threads: int = 1,  # pylint: disable=redefined-outer-name
     upload_url: Optional[str] = None,
     mode: str = "predict",
     is_build: bool = False,
     await_explicit_shutdown: bool = False,  # pylint: disable=redefined-outer-name
-) -> MyFastAPI:
-    app = MyFastAPI(  # pylint: disable=redefined-outer-name
-        title="Cog",  # TODO: mention model name?
```

Comment:
Where was this title used? The generated OpenAPI?

Reply:
Yes, I think so. It's now down here: https://github.com/replicate/cog/pull/1866/files#diff-424c9aeb9bb84b2114a86a5fdf84f4e2cbecf9797b03e6866caa03bd07e53b7cR124